api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -16,15 +16,15 @@
 using TNO.Kafka;
 using TNO.API.Config;
 using TNO.Kafka.Models;
-using TNO.Keycloak;
 using System.Net.Mime;
 
 namespace TNO.API.Areas.Editor.Controllers;
 
 /// <summary>
 /// ContentController class, provides Content endpoints for the api.
 /// </summary>
-[ClientRoleAuthorize(ClientRole.Editor)]
+// [ClientRoleAuthorize(ClientRole.Editor)]
+[Authorize]
```

Comment:
If a user is authenticated we allow them access to these endpoints.  Not ideal, but presently a workaround to get everything again.  I'll create a custom authorization tag for the service account when I have time.